.github/workflows/diff_report.yml

```diff
@@ -106,19 +106,20 @@ jobs:
           PATCH_CONFIG_LINK: ${{ needs.parse_body.outputs.patch_config_link }}
           LINK_FROM_PR: ${{ needs.parse_body.outputs.projects_link }}
         run: |
-         LINK="${LINK_FROM_PR:-$DEFAULT_PROJECTS_LINK}"
-         wget -q "$LINK" -O project.properties
-         if [ -n "$NEW_MODULE_CONFIG_LINK" ]; then
-           wget -q "$NEW_MODULE_CONFIG_LINK" -O new_module_config.xml
-         fi
-
-         if [ -n "$DIFF_CONFIG_LINK" ]; then
-           wget -q "$DIFF_CONFIG_LINK" -O diff_config.xml
-         fi
-
-         if [ -n "$PATCH_CONFIG_LINK" ]; then
-           wget -q "$PATCH_CONFIG_LINK" -O patch_config.xml
-         fi
+          LINK="${LINK_FROM_PR:-$DEFAULT_PROJECTS_LINK}"
```

Comment:
Indentation was -1 for this block, not sure if this had something to do with problems or just one of the links below.

Reply:
It should not, we have `|` that is multiline marker

Reply:
Multiline marker does count on increased indentation in most interpreters, but it should have been good since it was indented more than it's key. Better to fix it either way. This file is too complex for most interpreters, I think this is why yamllint failed to find bad indentation.